compiler/modules/bir.sexpr/serialize.bal

```diff
@@ -97,14 +97,20 @@ function fromBasicBlock(FuncSerializeContext sc, bir:BasicBlock block, bir:File
 
 function formInsn(FuncSerializeContext sc, bir:Insn insn, bir:File file) returns Insn {
     if insn is bir:CallInsn {
-        bir:FunctionRef func =  insn.func;
+        bir:FunctionOperand funcOperand = insn.operands[0];
+        boolean restArgIsList = insn.restArgIsList;
```

Comment:
Check if we can use pattern marching to de-structure.

Reply:
Fixed